utils/guidelines_xslt/odd2html.xsl

```diff
@@ -130,6 +130,13 @@
     </xd:doc>
     <xsl:param name="basedir" select="''" as="xs:string"/>
     
+    <xd:doc>
+        <xd:desc>
+            <xd:p>Whether Contributors should be retrieved live from GitHub. Please consider that this may result in excessive API calls.</xd:p>
+        </xd:desc>
+    </xd:doc>
+    <xsl:param name="retrieve-contributors" select="'false'" as="xs:string"/>
```

Comment:
Why not 
```suggestion
    <xsl:param name="retrieve-contributors" select="false()" as="xs:boolean"/>
```

Reply:
as commented in https://github.com/music-encoding/music-encoding/pull/1471#issuecomment-2128880033 this solution is preferable.